Doppler.HtmlEditorApi/Controllers/TemplatesController.cs

```diff
@@ -126,9 +126,23 @@ public TemplatesController(ITemplateRepository templateRepository)
 
         [Authorize(Policies.OwnResourceOrSuperUser)]
         [HttpPost("/accounts/{accountName}/templates")]
-        public Task<IResult> CreateTemplate(string accountName, Template templateModel)
+        public async Task<CreatedAtRoute<ResourceCreated>> CreateTemplate(string accountName, Template template)
         {
-            throw new NotImplementedException();
+            // TODO: add restriction for idTemplate diff to 0, return BadRequest
```

Comment:
What do you think about implementing it before merging the PR?

Reply:
Ok I'm agree, I will add a new commit with this implementation.

Reply:
Checking the code I seeing that is unnecessary add a **_BadRequest_** response when the _TemplateId is 0_ because the input model of _Template_ not allow the id and the repository (_repositoryTemplate_) **have a validation for templateId == 0**, so I prefer keep the current validation in the repository and remove the **TODO: message**